specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Product.json

```diff
@@ -0,0 +1,237 @@
+                        "$ref": "./examples/Product/Get.json"
+                    }
+                },
+                "description": "Return product name.",
+                "tags": [
+                    "Products"
+                ],
+                "operationId": "Products_Get",
+                "parameters": [
+                    {
+                        "$ref": "AzureBridge.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ActivationNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ProductNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ProductResource"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found"
+                    }
+                },
+                "produces": [
+                    "application/json"
+                ],
+                "consumes": [
+                    "application/json"
+                ]
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/providers/Microsoft.AzureBridge.Admin/activations/{activationName}/products/{productName}/download": {
+            "post": {
+                "x-ms-examples": {
+                    "Return product name.": {
+                        "$ref": "./examples/Product/download.json"
+                    }
+                },
+                "description": "Downloads a product from azure marketplace.",
+                "tags": [
+                    "Products"
+                ],
+                "operationId": "Products_Download",
+                "parameters": [
+                    {
+                        "$ref": "AzureBridge.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ActivationNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ProductNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "202": {
```

Comment:
Same here. It should also have 200 or 204.

Reply:
Done.